src/main/java/ru/javawebinar/topjava/dao/MealDaoImplMem.java

```diff
@@ -0,0 +1,53 @@
+package ru.javawebinar.topjava.dao;
+
+import org.slf4j.Logger;
+import ru.javawebinar.topjava.model.Meal;
+import ru.javawebinar.topjava.util.MealsUtil;
+
+import java.time.LocalDateTime;
+import java.time.Month;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+public class MealDaoImplMem implements MealDao {
+    private static final Logger log = getLogger(MealDaoImplMem.class);
```

Comment:
static final values should be named like LOG or LOGGER (in upper case). They called 'Constant'